packages/calypso-color-schemes/src/shared/color-schemes/_nightfall.scss

```diff
@@ -108,23 +108,23 @@
 	--color-masterbar-toggle-drafts-editor-hover-background: var(--studio-gray-40);
 	--color-masterbar-toggle-drafts-editor-border: var(--studio-gray-10);
 
-	--color-sidebar-background: var(--studio-blue-80);
+	--color-sidebar-background: #02395c;
 	--color-sidebar-background-rgb: var(--studio-blue-80-rgb);
 	--color-sidebar-border: var(--studio-blue-90);
-	--color-sidebar-text: var(--studio-blue-5);
+	--color-sidebar-text: #bbe0fa;
 	--color-sidebar-text-rgb: var(--studio-blue-5-rgb);
-	--color-sidebar-text-alternative: var(--studio-blue-20);
+	--color-sidebar-text-alternative: #68b3e8;
 	--color-sidebar-gridicon-fill: var(--studio-blue-10);
 	--color-sidebar-menu-selected-background: var(--studio-blue-100);
 	--color-sidebar-menu-selected-background-rgb: var(--studio-blue-100-rgb);
 	--color-sidebar-menu-selected-text: var(--studio-white);
 	--color-sidebar-menu-selected-text-rgb: var(--studio-white-rgb);
-	--color-sidebar-menu-hover-background: var(--studio-blue-70);
+	--color-sidebar-menu-hover-background: #064b7b;
```

Comment:
is this a fix? [`--studio-blue-70` was `#044b7a`](https://github.com/Automattic/color-studio/pull/764/files?diff=unified&w=1#diff-a454ab39981381c132ee553093d01f476558bad46da1444c70c65528e883750bL32)